src/utils/showAnswerUtil.js

```diff
@@ -0,0 +1,26 @@
+import { appendLinks } from '../utils/appendLinksUtil.js';
+
+export const showAnswer = (answerElement, key, currentQuestion) => {
```

Comment:
Since this function is actually showing the correct answer, should we rename it as `showCorrectAnswer` ? What do you think?

Reply:
Of course ### showCorrectAnswer is better!